stable/distributed-tensorflow/README.md

```diff
@@ -76,11 +76,20 @@ chart and their default values.
 
 | Parameter | Description | Default |
 |-----------|-------------|---------|
-| `worker.image.repository` | TensorFlow Worker Server's image repository | `cheyang/distributed-tf` |
+| `worker.image.repository` | TensorFlow Worker Server's image repository | `dysproz/distributed-tf` |
 | `worker.image.tag` | TensorFlow Worker Server's image tag | `gpu` |
 | `worker.image.pullPolicy` | image pullPolicy for the  worker | `IfNotPresent` |
 | `worker.gpuCount` | Set the gpu to be allocated and allowed for the Pods | `0` |
-| `ps.image.repository` | TensorFlow Parameter Server's image repository | `cheyang/distributed-tf` |
-| `ps.image.tag` | TensorFlow Parameter Server's image tag | `1.6.0-gpu` |
+| `worker.env` | key-value environment variables | None |
+| `ps.image.repository` | TensorFlow Parameter Server's image repository | `dysproz/distributed-tf` |
+| `ps.image.tag` | TensorFlow Parameter Server's image tag | `1.7.0-gpu` |
 | `ps.image.pullPolicy` | image pullPolicy for the  ps | `IfNotPresent` |
-
+| `ps.env` | key-value environment variables | None |
+| `volumes` | List of volumes defined in cluster (in standard k8s format) | host path to */tmp/mnist* |
+| `volumeMounts` | Volumes mounted into Pods (in standard k8s format) | host path volume mounted into */tmp/mnist-log* |
+| `hyperparams.batchsize` | batch size | `20` |
+| `hyperparams.learningrate` | learning rate | `0.001` |
+| `hyperparams.trainsteps` | train steps | `0` (continious run) |
```

Comment:
please run spell check, I believe it should be `continuous`

Reply:
Done